srcpkgs/dracut/files/kernel-uefi-hook-postrm

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+#
+# Kernel post-remove hook for dracut EFI bundle.
+#
+# Arguments passed to this script: $1 pkgname, $2 version.
+#
+PKGNAME="$1"
+VERSION="$2"
+
+. "${ROOTDIR}/etc/default/dracut-uefi-hook"
+if [ "x${CREATE_UEFI_BUNDLES}" != x1 ]; then
+	exit 0
+fi
+
+: "${UEFI_BUNDLE_DIR:=boot/efi/EFI/Void}"
```

Comment:
Lowercase `V` in `Void`.
Since grub will create this very same directory with lowercase `void`.
I'm not sure what they will do if they're conflicting now.
And I couldn't say about their future.